### DISCUSSION TITLE
Help with official deployment steps on Cloudflare Pages

### DISCUSSION THREAD

**Discussion by pataquets** [marp-team/marp#476] [Q&A]
Hi there!
First and foremost, thanks for sharing this awesome app. It's incredibly powerful.
Even with my limited JS/Jamstack/front experience, I've been able to deploy a presentation on [Cloudflare Pages](https://pages.cloudflare.com/), using CF's free plan.
So, I wondered if CF could add MARP as an officially supported [framework in their  "Cloudflare Pages" docs](https://developers.cloudflare.com/pages/framework-guides/) and filed cloudflare/cloudflare-docs#7579 and even got an answer from CF's team.
However, I can't find any relevant docs which might help in answering their requirements or if they even exist. Note, thou, it might be my limited knowledge, as I barely could publish the presentation myself.
Anyone more knowledgeable could take a look at it? It would be great for MARP to be officially supported in such an awesome product and I think it will give it a boost in visibility.
Thanks.

**Comment by yhatt** [comment]
Marp tools that make up the ecosystem are [documented at each repository in detail](https://github.com/marp-team/marp#marp-family).

[Marp CLI](https://github.com/marp-team/marp-cli) has been designed as a converter for simple Markdown file(s), just like `abc.md` to `abc.html`, and does not limit a workflow to a specific configuration. If a specific configuration was defined as an official workflow, there is a potential risk that it leads to the user's misperception that the official workflow is the only way to use Marp effectively. It's a reason why [marp-cli-example](https://github.com/yhatt/marp-cli-example) is hosted by the maintainer, not the Marp team.

We think it's sufficient to configure Cloudflare Pages according to [existing Cloudflare guidance](https://developers.cloudflare.com/pages/framework-guides/deploy-anything/) and the output result of Marp CLI. Of course, suggesting the example of the workflow for Cloudflare Pages by the Marp community is welcome :)
